src/rpc/rpc_tcp_server.js

```diff
@@ -26,7 +25,9 @@ class RpcTcpServer extends EventEmitter {
             net.createServer(tcp_conn => this._on_tcp_conn(tcp_conn));
         this.server.on('close', err => {
             dbg.log0('on close:', err);
-            this.emit('error', new Error('TCP SERVER CLOSED'));
+            // emitting this as error is not desirable since no one is listening and it gives Uncaught Error.
+            // this happens on test_rpc as after each test we call disconnect. 
+            // this.emit('error', new Error('TCP SERVER CLOSED'));
```

Comment:
```suggestion
            // emitting this as an error is not desirable since no one is listening and it gives Uncaught Error.
            // This happens on test_rpc as after each test we call disconnect. 
            // this.emit('error', new Error('TCP SERVER CLOSED'));
```